### DISCUSSION TITLE
Julia output not displaying in quarto

### DISCUSSION THREAD

**Discussion by DomDF** [quarto-dev/quarto-cli#1980] [Q&A] [answered]
I have an existing query in the RStudio community page, but have been recommended to ask here too. 
If anyone can provide a solution, I will update on this page too: https://community.rstudio.com/t/julia-output-not-displaying-in-quarto/144462

------------

I am writing a quarto document, which includes code chunks in R, Python and Julia.

The output from the Python and R chunks is displaying correctly. However, while the code for the Julia chunks is displaying, the output is not - see the below image:

<img width="692" alt="image" src="https://user-images.githubusercontent.com/34099039/185512471-14dd6ed5-df8e-4782-b9c8-67e7ac07aa33.png">

My YAML header is included below:

```
format:
  html:
    theme: spacelab
    html-math-mathod: mathjax
    toc: true
    toc-location: right
    number-sections: true
    code-copy: true
    code-fold: true
    code-tools: true
    monofont: Fira Code
execute:
  message: false
  warning: false
bibliography: references.bib
```

Please advise on how I can see the output of my Julia code chunks in this document too.

Many thanks

**Comment by cscheid** [comment, marked as answer]
Hi, thanks for the report. Can you file a github issue on this? Specifically, we need a minimal, self-contained .qmd where that issue happens.

**Reply by DomDF** [reply]
The issue was resolved after installing the latest versions of Julia, Quarto and RStudio. I should have done this one at a time to help identify the issue, but this is what I would recommend to anyone experiencing this issue.
